### PR TITLE
fix crash when sqs queue is just deleted and still appears in listing

### PR DESCRIPTION
**Pull request by moadibfr**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | yes
| 🚀 New feature?   | no
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | 
| ❓ Documentation  | no 

## Description
fix crash when sqs queue is just deleted and still appears in listing

### CODE REVIEW

File: pkg/remote/aws/sqs_queue_details_fetcher.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+func (r *SQSQueueDetailsFetcher) ReadDetails(res resource.Resource) (resource.Resource, error) {
+	ctyVal, err := r.reader.ReadResource(terraform.ReadResourceArgs{
+		ID: res.TerraformId(),
+		Ty: aws.AwsSqsQueueResourceType,
+	})
+	if err != nil {
+		if strings.Contains(err.Error(), "NonExistentQueue") {
+			logrus.Debugf("Ignoring %+v", err)
```
Comment (eliecharra) [inline pkg/remote/aws/sqs_queue_details_fetcher.go:31]: Maybe a more explicit message with additional context could be more usefull in debug traces. We did it previously for EC2 instances

```go
logrus.WithFields(logrus.Fields{
			"id": res.TerraformId(),
		}).Debug("Instance read returned nil (instance may be terminated), ignoring ...")
```

File: pkg/remote/aws/sqs_queue_details_fetcher.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		deserializer: deserializer,
+	}
+}
+
+func (r *SQSQueueDetailsFetcher) ReadDetails(res resource.Resource) (resource.Resource, error) {
+	ctyVal, err := r.reader.ReadResource(terraform.ReadResourceArgs{
+		ID: res.TerraformId(),
+		Ty: aws.AwsSqsQueueResourceType,
+	})
+	if err != nil {
```
Comment (eliecharra) [inline pkg/remote/aws/sqs_queue_details_fetcher.go:29]: We should add a test to cover this case

**Review by eliecharra — CHANGES_REQUESTED** [review]

File: pkg/remote/aws/sqs_queue_details_fetcher.go (hunk 2 above)
Comment (moadibfr) [reply pkg/remote/aws/sqs_queue_details_fetcher.go:29]: Yeah I'd like to but we don't have test for details fetcher with mock and I couldn't reproduce this one to have goldenfile...

Comment (eliecharra) [reply pkg/remote/aws/sqs_queue_details_fetcher.go:29]: 🆗

**Review by eliecharra — APPROVED** [review]

**Pull request merged**